app/views/landingpage/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<a href="/">Home</a>
+<a href="/books">Shelf</a>
```

Comment:
Adds what we needed to get rolling.

We'll be sure to fill this out as we continue.